e2e-tests/cypress/tests/integration/channels/messaging/message_bullets_spec.js

```diff
@@ -39,7 +39,7 @@ describe('Message', () => {
                 const expectedSize = '13.5px';
 
                 cy.get('p').first().should('have.text', 'This is a normal sentence.').and('have.css', 'font-size', expectedSize);
-                cy.get('ol li').first().should('have.text', 'this is point 1\nthis is a bullet under 1').and('have.css', 'font-size', expectedSize);
+                cy.get('ol li').first().should('have.text', 'this is point 1this is a bullet under 1').and('have.css', 'font-size', expectedSize);
```

Comment:
This looks weird (the two strings concatenated like that). Is this expected?

Reply:
@larkox its this part, some how it doesnt have `\n`
will this be a defect?
<img width="314" alt="image" src="https://github.com/mattermost/mattermost/assets/15979783/950dfcb8-d1e4-4d84-b43c-4f66141a0601">

Reply:
Looks great like this :smile: